src/main/java/io/strimzi/kafka/quotas/StaticQuotaConfig.java

```diff
@@ -159,6 +159,16 @@ private static ConfigDef.LambdaValidator iso8601DurationValidator() {
         }, () -> "Should be a valid iso8601 duration string like PT5M");
     }
 
+    /**
+     * Get the broker id of the current broker.
+     * @return the specified broker id or -1 if not defined.
+     */
+    public String getBrokerId() {
+        //Arguably in-efficient to look up the sys prop if we don't need it, but it reads better and is invoked rarely
+        final String brokerIdFromSysProps = System.getProperty("broker.id", "-1");
```

Comment:
It's not clear to me why we look up a system property if we can't find it in the config.

Reply:
As the broker id can be specified as a system property as well IIRC so it looks up the system property as a fallback incase broker.id wasn't in the `original` (pre interpolation) set of config properties.

Reply:
As the broker id can be specified as a system property as well IIRC so it looks up the system property as a fallback incase broker.id wasn't in the original (pre interpolation) set of config properties. 

Its of debatable value as in all Strimzi deployments the broker.id would be in the originals config.

Reply:
> As the broker id can be specified as a system property as well IIRC

Can it?

Reply:
Apparently not, so I've been a member of a cargo cult for a long time.

Removed in dc1011837c8c51e7781086c252cc4aedcec2021a